message_definitions/v1.0/ardupilotmega.xml

```diff
@@ -120,6 +120,17 @@
                 <param index="6">Empty</param>
                 <param index="7">Empty</param>
             </entry>
+            
+            <entry name="MAV_CMD_ACCELCAL_VEHICLE_POS" value="42429">
+                <description>Used when doing accelerometer calibration. When sent to the GCS tells it what position to put the vehicle in. When sent to the vehicle says what position the vehicle is in.</description>
+                <param index="1">1: Level, 2: Left, 3: Right, 4: Nose down, 5: Nose up, 6: Back</param>
```

Comment:
I'll do it, but I think there's no precedent to create an enum just for the values in a param in MAV_CMD.